Source/WebCore/platform/graphics/PathSegmentData.h

```diff
@@ -165,6 +165,28 @@ struct PathArc {
 
 WEBCORE_EXPORT WTF::TextStream& operator<<(WTF::TextStream&, const PathArc&);
 
+struct PathClosedArc {
```

Comment:
Can't this structure contain a single member of type `PathArc' instead of repeating the same data members here?